inputs/worker_inner:4.json

```diff
@@ -0,0 +1,136 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "repos": [],
+        "from_task_id": "{{FROM_TASK_ID}}"
+      }
+    },
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "name": "add_labels_in_dockerfile",
+      "args": {
+        "labels": "{{IMPLICIT_LABELS}}"
+      }
+    },
+    {
+      "name": "change_from_in_dockerfile"
+    },
+    {
+      "name": "add_help",
+      "required": false
+    },
+    {
+      "name": "add_dockerfile"
+    },
+    {
+      "name": "distgit_fetch_artefacts",
+      "args": {
+        "command": "{{SOURCES_COMMAND}}"
+      }
+    },
+    {
+      "args": {
+        "koji_root": "{{KOJI_ROOT}}",
+        "koji_hub": "{{KOJI_HUB}}"
+      },
+      "required": false,
+      "name": "fetch_maven_artifacts"
+    },
+    {
+      "name": "koji",
+      "args": {
+        "root": "{{KOJI_ROOT}}",
+        "target": "{{KOJI_TARGET}}",
+        "hub": "{{KOJI_HUB}}"
+      }
+    },
+    {
+      "name": "add_yum_repo_by_url",
+      "args": {
+        "repourls": []
+      }
+    },
+    {
+      "name": "inject_yum_repo"
+    },
+    {
+      "name": "distribution_scope"
+    }
+  ],
+  "prepublish_plugins": [
+    {
+      "name": "squash"
+    }
+  ],
+  "postbuild_plugins": [
+    {
+      "name": "all_rpm_packages",
+      "args": {
+        "image_id": "BUILT_IMAGE_ID"
+      }
+    },
+    {
+      "name": "tag_from_config",
+      "args": {
+        "tag_suffixes": "{{TAG_SUFFIXES}}"
+      }
+    },
+    {
+      "name": "tag_and_push",
+      "args": {
+        "registries": {
+          "{{REGISTRY_URI}}": { "insecure": true }
+        }
+      }
+    },
+    {
+      "name": "pulp_push",
+      "args": {
+        "pulp_registry_name": "{{PULP_REGISTRY_NAME}}",
+        "load_exported_image": true,
+        "dockpulp_loglevel": "INFO",
+        "sync": false
```

Comment:
This should be "publish", not "sync" (my fault, I wrote it up wrong originally).
http://osbs.readthedocs.io/en/latest/multiarch.html#pulp-push